scripts/manifest.yml

```diff
@@ -0,0 +1,30 @@
+---
+version: 1.0.0.0-rc1
+
+download_urls:
+  OpenSearch: https://artifacts.opensearch.org/snapshots/core/opensearch/1.0.0/opensearch-min-1.0.0-SNAPSHOT-linux-x64-186.tar.gz
```

Comment:
Consider structuring these in separate configuration files, or as a hierarchy like so:

```
products:
  opensearch:
    libs:
    plugins:
  opensearch-dashboards:
    plugins:
```

Reply:
+1

Reply:
Sure will make the change. This script only concentrates on opensearch. We have different script for dashboards. For manifest I think I'll restructure it as hierarchy so there is low maintenance. Let me know if you think otherwise

Reply:
Made the changes